### DISCUSSION TITLE
Trying to set consumer.offsetCommitIntervalMs: 10000, but it is not working.

### DISCUSSION THREAD

**Discussion by yogeshgoel001** [strimzi/strimzi-kafka-operator#10215] [General]
Trying to set consumer.offsetCommitIntervalMs: 10000, but it is not working.  I can still see that mirrormaker2 is taking 60 seconds to commit the offset number at the destination. How to resolve the issue.

**Comment by scholzj** [comment]
Duplicate of #10216
